src/CMakeLists.txt

```diff
@@ -69,6 +69,28 @@ file (GLOB xournalpp_SOURCES_RECURSE
   pdf/popplerapi/*.cpp
 )
 
+if (WIN32)
+  # generate .ico from .png
+  set (ICON_SIZES 16 32 48 256)
+  foreach (SIZE IN LISTS ICON_SIZES)
+    set (RESIZED_ICON ${PROJECT_BINARY_DIR}/xournalpp-icon-${SIZE}.png)
+    add_custom_command (
+	  OUTPUT ${RESIZED_ICON}
+	  COMMAND rsvg-convert ${PROJECT_SOURCE_DIR}/ui/pixmaps/com.github.xournalpp.xournalpp.svg -o ${RESIZED_ICON}
```

Comment:
You should specify width and height here. Otherwise all icons are identic, 128x128 pixel.

Reply:
I fixed this.